<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pbtn_connect">
          <property name="text">
           <string>绑定端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_disconnect">
          <property name="text">
           <string>解除绑定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_clear">
          <property name="text">
           <string>清空文本框</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_exit">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_findip">
          <property name="text">
           <string>本地地址</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>绑定端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bind_port">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>目标地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ip">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>目标端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="global_port">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QTextEdit" name="textEdit_display"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_message"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pbtn_send">
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pbtn_broadcast">
        <property name="text">
         <string>广播消息</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
